deno.json

```diff
@@ -1,7 +1,7 @@
 {
   "tasks": {
     "chel": "deno task build && deno run --allow-net --allow-read=. --allow-write=. build/main.js",
-    "lint": "standard ./src",
+    "lint": "npx ts-standard ./src ./scripts --fix",
```

Comment:
This `--fix` needs to be removed as it can modify code unintentionally.

Reply:
https://github.com/okTurtles/chel/pull/49/commits/224221d23de5206ac4a8a20e85766b104845b546